com/paypal/Perfection.java

```diff
@@ -0,0 +1,42 @@
+package com.paypal;
+
+public class Perfection {
+
+    private static Perfection perf;
+
+    public synchronized static Perfection getPerf() {
```

Comment:
given that the other two methods are nto abreviated, we should use getPerfection for consistency